src/index.ts

```diff
@@ -336,4 +336,23 @@ export class HTTPSnippet {
     const results = this.requests.map(request => convert(request, options));
     return results;
   }
+
+  installation(targetId: TargetId, clientId?: ClientId, options?: any): (string | false)[] | false {
+    if (!this.initCalled) {
+      this.init();
+    }
+
+    if (!options && clientId) {
+      options = clientId;
+    }
+
+    const target = targets[targetId];
+    if (!target) {
+      return false;
+    }
+
+    const { installation } = target.clientsById[clientId || target.info.default];
+    const results = this.requests.map(request => (installation ? installation(request, options) : false));
```

Comment:
i don't totally follow why this is an array. is it because you can pass several HAR files into this library?

Reply:
yeah that's correct. it's not something we've ever done and is just holdoverr from upstream